### ISSUE TITLE
MapInfo Tab don't opened

### ISSUE THREAD

**Issue by Bescu** [rburhum/arcgis-ogr#4]
Hello,

I also tried to add somes MapInfo TAB with the plugin but It didn't work : the table is shown in the layer selection space of the plugin, but when I click ok the "OK button", it doesn't do nothing.

**Comment by rburhum** [comment]
Can I get a sample MapInfo file that reproduces this? It will help with debugging.

**Comment by Bescu** [comment]
Yes of course, the sample MapInfo file is here : http://dl.free.fr/nLL0GYybn

**Comment by rburhum** [comment]
Looks like it is also UTF8 related.

**Comment by rburhum** [comment]
In addition, even though I sanitize those fields, it looks like ogr_layer.GetGeomType() is returning wkbUnknown. I will have to debug the bindings, too. I'll let you know what I find.

**Comment by Bescu** [comment]
Ok! I see! Thank you!

**Comment by rburhum** [comment]
I fixed the UTF-8 related stuff. Nevertheless, I am unsure how to go about "fixing" the issue with a mix of geometries.
Sadly, ArcGIS only supports one geometry type per layer. I see a few options, like a manual settings that "forces" a certain geometry type. This could trick ArcGIS into thinking it was that geometry type.

For example, in your case, I would trick the layer into thinking it was only polygons. Then I would drop any other geometry type from the result set. This would in theory work. Another option is to add that behavior directly to GDAL/OGR, then all the other clients could benefit from something like that - although I am still unsure of the utility of that.
